README.md

```diff
@@ -1,2 +1,132 @@
-# final-capstone-frontend
-Final capstone frontend
+# EASY BOOKING APP
```

Comment:
- [ ]  I was able to run the frontend `npm start` with warnings and was able to get to the welcome screen (beautiful screen).  However, since I couldn't create the database, I wasn't able to go any further to review the functionality of the app.  Please, refer to the backend repo for the review comments.  On the next review when this issue is tackled, we will check the functionality of your app.

These are de compiler's warnings:

![image](https://user-images.githubusercontent.com/57421823/200047683-be09baac-c855-4ff7-a850-b6f7ae741742.png)